src/main/java/bowling/domain/Frame.java

```diff
@@ -1,62 +1,79 @@
 package bowling.domain;
 
-import bowling.dto.FrameDto;
-
 import java.util.ArrayList;
 import java.util.List;
 
-class Frame {
+public class Frame {
     private static final int SHOT_LIMIT = 2;
 
     private final ShotScores shotScores;
-    private final boolean hasBonus;
+    private final boolean islast;
```

Comment:
멤버 변수 이름은 lowerCamelCase로 작성합니다. 아래의 Google Java Style Guide를 참고해 주세요.

> Non-constant field names (static or otherwise) are written in lowerCamelCase.

Reply:
🆗